src/language/class-modifiers.md

```diff
@@ -202,9 +202,8 @@ To create a known, enumerable set of subtypes, use the `sealed` modifier.
 This allows you to create a switch over those subtypes that is statically ensured
 to be [_exhaustive_][exhaustive].
 
-The `sealed` modifier prevents a class from being extended or implemented outside
-of its own library. Sealed classes also prevent construction,
-and are therefore implicitly [abstract](#abstract).
+The `sealed` modifier prevents a class from being extended or
+implemented outside of its own library.
```

Comment:
This is different to the above. Arguably with this text a sealed class can be instantiated inside its library. Maybe split into two seperate sentences. Can't be extended outside of its library. Can't be instantiated directly.

Reply:
Yeah this mention was moved below the code block to extend that discussion. But now that you say that, it does seem a bit far away for an important concept. 

I'll move that back up for now. Thanks!

Reply:
Okay, I moved it back up, should cover that case fine now :)